.ci_support/linux_64_cuda_compiler_version10.2cudnn7.yaml

```diff
@@ -12,11 +12,14 @@ cuda_compiler:
 - nvcc
 cuda_compiler_version:
 - '10.2'
+cudnn:
+- '7'
```

Comment:
Hm, I'm not sure why `conda-smithy` did this. I thought these yaml files in .ci_support/ should be independent of cudnn, since this is the recipe for cudnn.

Reply:
Because there are a few keys bundled together, including `cudnn` which IMHO was a bad decision. See https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/1554.